test/integration/ggrc/access_control/rbac_factories/cycle_task.py

```diff
@@ -204,3 +211,39 @@ def restore(self):
     """Restore Cycle Task."""
     cycle_task = all_models.CycleTaskGroupObjectTask.query.first()
     return self.api.put(cycle_task, {"status": "Assigned"})
+
+  def assign(self):
+    """Assign Cycle Task."""
+    with factories.single_commit():
+      user_obj = factories.PersonFactory()
+      permission_factories.UserRoleFactory(
+          role=all_models.Role.query.filter_by(
+              name="Reader"
+          ).first(),
+          person=user_obj
+      )
+    acl_role_obj = all_models.AccessControlRole.query.filter_by(
+        object_type='Workflow',
+    ).first()
+    cycle_task = all_models.CycleTaskGroupObjectTask.query.get(
+        self.cycle_task_id
+    )
+    return self.api.put(
+        cycle_task,
+        {
+            "ac_role_id": acl_role_obj.id,
+            "person": {"id": user_obj.id, "type": "Person"}
+        }
+    )
```

Comment:
Should not it be like the following instead? Please correct me if I am wrong.
```suggestion
    return self.api.put(
        cycle_task,
        {
            "access_control_list": {
                "ac_role_id": acl_role_obj.id,
                "person": {"id": user_obj.id, "type": "Person"},
            }
        }
    )
```

Reply:
The same if true for all other `assign` methods.